src/Fabulous.XamarinForms/Views/Layouts/Frame.fs

```diff
@@ -0,0 +1,51 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Fabulous.XamarinForms
+open Xamarin.Forms
+
+type IFrame =
+    inherit ITemplatedView
```

Comment:
Actually this one inherits from ContentView

Reply:
Updated 😀